src/components/user-settings.tsx

```diff
@@ -33,7 +33,9 @@ const UserInfo = ( {
 			<div className="flex w-full items-center gap-3">
 				<Button
 					onClick={ () => getIpcApi().openURL( WPCOM_PROFILE_URL ) }
-					aria-label={ __( 'Profile link' ) }
+					aria-label={ __( 'Edit Profile' ) }
+					label={ __( 'Edit Profile' ) }
```

Comment:
Not sure if it should be `Edit profile` or `Edit Profile` considering that before it was `Profile link`?

Reply:
Good point, let's stick with the 'Edit profile' casing in all cases.

Reply:
I was considering both options but then picked `Edit Profile` and `Open Settings`. Regardless, using `Edit profile` and `Open settings` sounds good to me as well, so I will apply the changes as you suggested. 🙂

Reply:
I have now addressed the change in https://github.com/Automattic/studio/pull/648/commits/791c4b0c3a290d2414990ceebdea212a3cc792f0.